### PR TITLE
Add review talks functionality

### PR DESCRIPTION
**Pull request by korzonkiee**

### CODE REVIEW

File: lib/rate/bloc/rate_bloc.dart [hunk 1]
```diff
@@ -32,22 +36,51 @@ class RateBloc extends Bloc<RateEvent, RateState> {
...
+        yield RatingTalkToEarlyErrorState();
+      }
+    } catch (e, s) {
+      logger.errorException(e, s);
+      yield RatingTalkErrorState();
     }
   }
+
+  bool canRateTalk(Talk talk) =>
+      DateTime.now().isAfter(talk.endTime.subtract(Duration(minutes: 5)));
```
Comment (orestesgaolin) [inline lib/rate/bloc/rate_bloc.dart:85]: Can we make it configurable from remote config maybe? Maybe in another PR...

File: lib/rate/widgets/write_review_modal_bottom_sheet.dart [hunk 2]
```diff
@@ -0,0 +1,104 @@
...
+                  borderSide: BorderSide(color: Theme.of(context).accentColor),
+                ),
+              ),
+            ),
+            Row(
+              mainAxisAlignment: MainAxisAlignment.spaceBetween,
+              children: <Widget>[
+                Expanded(
+                  child: Text(
+                    "Review is visible only to the speaker.",
```
Comment (orestesgaolin) [inline lib/rate/widgets/write_review_modal_bottom_sheet.dart]: _Review will be visible only to the speaker_

**Review by orestesgaolin — APPROVED** [review]
LGTM

File: lib/rate/bloc/rate_bloc.dart (hunk 1 above)
Comment (korzonkiee) [reply lib/rate/bloc/rate_bloc.dart:85]: Sure. I'll make another PR to fix that.

**Pull request merged**